src/elements/content-preview/preview-header/PreviewHeaderComponent.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * @flow
+ * @file Preview header container component
```

Comment:
This doesn't appear to be a container, since it doesn't connect to any apis/data sources and contains markup.